samples/widgets/spinbtn.cpp

```diff
@@ -459,14 +459,8 @@ void SpinBtnWidgetsPage::OnButtonSetMinAndMax(wxCommandEvent& WXUNUSED(event))
     m_min = minNew;
     m_max = maxNew;
     wxString smax('9', m_textMax->GetValue().length());
-    wxSize
-      size = m_spinctrl->GetSizeFromTextSize(m_spinctrl->GetTextExtent(smax));
-
-    m_spinctrl->SetMinSize(size);
-    m_spinctrl->SetSize(size);
-
```

Comment:
Sorry, I don't really understand the meaning of this change, could you please explain?